jwt.d.ts

```diff
@@ -140,7 +141,7 @@ export interface FastifyJwtDecodeOptions {
 
 declare module 'fastify' {
   interface FastifyInstance {
-    jwt: JWT
+    jwt: JWT & { [namespace: string]: JWT }

```

Comment:
what does this mean in practice?

Reply:
It does not serve a good purpose, so removing it. I also check with typescript documentation. it was mistake, removing it

Reply:
I have updated, can you check it ?